torch/_export/utils.py

```diff
@@ -1150,3 +1153,146 @@ def _compiling_state_context():
     finally:
         torch.compiler._is_compiling_flag = old_compiling_flag
         torch.compiler._is_exporting_flag = old_exporting_flag
+
+
+def register_module_as_pytree_input_node(
+    cls: Type[torch.nn.Module],
+    flatten_fn: Optional[FlattenFunc] = None,
```

Comment:
Looks like all these optional params are actually not used. Could we make this method simpler?